motr/st/utils/m0crate_st_workloads.yaml

```diff
@@ -35,7 +35,7 @@ WORKLOAD_SPEC:                   # Workload specification section
    WORKLOAD:                     # First Workload
       WORKLOAD_TYPE: 1           # Workload type, 0->Index, 1->IO
       WORKLOAD_SEED: tstamp      # SEED to the random number generator.
-      OPCODE: 2                  # CREATE/OPEN/WRITE/READ/DELETE/POPULATE/CLEANUP=(0,1,2,3,4,5,6).
+      OPCODE: 2                  # CREATE/OPEN/WRITE/READ/DELETE/POPULATE/CLEANUP/READ_ONLY=(0,1,2,3,4,5,6,7).
```

Comment:
Better to keep cleanup as a last parameter for more readability

Reply:
In enum m0_operations{ }, I added READ_ONLY as last parameter because adding in-between would change opcode of subsequent operations (CLEANUP). and that needs to be change other places as well. To avoid this I added it to last as similarly into workload.yaml file.